spec/spec_helper.rb

```diff
@@ -21,13 +21,13 @@
 require_relative '../framework/modules/modules'
```

Comment:
rename 
require_relative '../framework/helpers/config_reader'
to
require_relative '../framework/helpers/config_helper'

Reply:
Fixed